### DISCUSSION TITLE
V1 on readme. Sunset in a few weeks due to high costs

### DISCUSSION THREAD

**Discussion by hafidzaini** [acheong08/ChatGPT#1071] [Q&A] [answered]
Does it mean, V1 using proxy will be shut down?
if it will, can we get the proxy source code so we can deploy to our own server?

**Comment by acheong08** [comment, marked as answer]
It will be open source once it sunsets

**Reply by acheong08** [reply]
<details> <summary></summary> dGhlIGJpbmFyeSBpcyBhbHNvIGF2YWlsYWJsZSBhdDog aHR0cHM6Ly9naXRodWIuY29tL1J1bmF3YXlWUE4vUFJPWC90cmVlL3RlbXAy  </details>

**Reply by hafidzaini** [reply]
thank you very much mate 🙏

**Comment by PlutoniaX** [comment]
will we be able to host our own proxy without being banned by OpenAI? $5 per month unlimited usage still is very worth it even compared to the reduced cost of ChatGPT API.

**Reply by hafidzaini** [reply]
yeah, might be better if @acheong08 provide this kind of service tho 🤔

**Reply by acheong08** [reply]
https://github.com/acheong08/ChatGPT/discussions/1081

**Reply by acheong08** [reply]
Oh it's gone. The bypass got killed

**Reply by hafidzaini** [reply]
i mean providing proxy service with monthly charge.
i dont mind to subscribe tho, rather than build and managing my own proxy server 😂🙏
